scripts/heroku-exec

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
Ancient macOS `/bin/bash` gives an error:

```
./scripts/heroku-exec: line 42: docker_args[@]: unbound variable
```

Homebrew-installed bash is 👍 but needs this shebang:

```suggestion
#!/usr/bin/env bash
```

Reply:
👀 https://github.com/nextstrain/ingest/pull/17#discussion_r1312013780

Reply:
Thanks, I forgot about that conversation (bad memory)! If there's a simple workaround that works for Bash 3.2 and above, better to use that.

Reply:
17a85d7e